contacts/contact_custom_report.sql

```diff
@@ -536,8 +536,7 @@ FROM student_session_aff AS enrollments
   LEFT JOIN emergency_contact_2 ON emergency_contact_2.student_id = enrollments.student_id
 
 WHERE
-  -- get students enrolled in the current academic year
-  sessions.academic_year = 2018
```

Comment:
This was redundant with the enrollment start and end date statement and made it such that we would have to update it next year.

Reply:
Sounds good. The counts still match the search 2.0 counts after this?

Reply:
Yep